app/helpers/stats_helper.rb

```
@@ -0,0 +1,233 @@
+    end
+
+    grouped_by_country.transform_values!(&:uniq)
+
+    {
+      countries_count: data[:countries].count,
+      cities_count: data[:cities].count,
+      grouped_by_country: grouped_by_country.transform_values(&:sort).sort.to_h,
+      year: year,
+      modal_id: "countries_cities_modal_#{year}"
+    }
+  end
```

RuboCop (1.76.1)

[convention] Assignment Branch Condition size for `countries_and_cities_stat_for_year` is too high. [<11, 52, 14> 54.96/17]

(Metrics/AbcSize)